frontend/assets/css/prime_origin.scss

```diff
@@ -8,18 +8,18 @@
   font-family: "Inter var", sans-serif;
   font-feature-settings: "cv02", "cv03", "cv04", "cv11";
   font-variation-settings: normal;
-  --font-family:"Inter var", sans-serif;
```

Comment:
would have preferred to keep this file untouched and add it to an ignore list. So it would be easier to compare when Primeview changes something to check how the styles changed.

Reply:
true